defaults

```diff
@@ -35,6 +35,15 @@ BACKUPS_INTERVAL=${BACKUPS_INTERVAL:-315360000}
 BACKUPS_CRON=${BACKUPS_CRON-0 * * * *}
 BACKUPS_DIRECTORY=${BACKUPS_DIRECTORY:-/config/backups}
 BACKUPS_MAX_AGE=${BACKUPS_MAX_AGE:-3}
+# Enable SFTP Copy of Backup zip
+BACKUPS_SFTP_ENABLE=${BACKUPS_SFTP_ENABLE:-0}
+[ "$BACKUPS_SFTP_ENABLE" = true ] && BACKUPS_SFTP_ENABLE=1
```

Comment:
I don't understand this logic. Why not just check for `[ "$BACKUPS_SFTP_ENABLE" = true ]` in all the lines below where it currently checks for `[ "$BACKUPS_SFTP_ENABLE" = 1 ]`?